opentripplanner-geocoder/src/main/resources/org/opentripplanner/geocoder/application-context.xml

```diff
@@ -174,21 +202,31 @@
 				<entry key="(RZE) Zeta Hall" value="28.06670 -82.41264" />
 
 				<entry key="(SCA) Science Center" value="28.0605 -82.4158" />
+				<entry key="(SCD) Softball Stadium" value="28.05796 -82.40445" />
+				<entry key="(SEC) SE Chiller Plant" value="28.05923 -82.40744" />
+				<entry key="(SHR) Shriners Hospital for Children" value="28.06163 -82.42278" />
 				<entry key="(SHS) Student Health Services Building" value="28.0636 -82.4119" />
 				<entry key="(SOC) Social Science Building" value="28.0615 -82.4092" />
-				<entry key="(SPS) Corbett Soccer Park Stadium" value="28.06007 -82.40383" />
+				<entry key="(SPS) Soccer Stadium" value="28.06007 -82.40383" />
+				<entry key="(SRB) Stabile Research Building" value="28.06565 -82.42144" />
+				<entry key="(STA) Stadium" value="28.06302 -82.40670" />
+				<entry key="(SUN) Sun Dome" value="28.05917 -82.40654" />
 				<entry key="(SVC) Student Services Building" value="28.0625 -82.4124" />
 
 				<entry key="(TAR) Theatre Centre" value="28.0640 -82.4145" />
 				<entry key="(TAT) Theatre Auditorium - Theatre I" value="28.0636 -82.4145" />
 				<entry key="(THR) Theatre II" value="28.0636 -82.4149" />
 				<entry key="(TVB) WUSF Television Station" value="28.0623 -82.4118" />
 
+				<entry key="(UDI) University Diagnostic Institute (UDI)" value="28.05721 -82.42123  " />

```

Comment:
I think this looks good.   Were you able to test that the geocoder is able to parse and locate the new/changed locations here?

The only potential concern that I see is this line because of the extra whitespace at the end, though I think it is probably fine.

Also, for the places with different latlngs, those came from the USF maps, or something else?  One thing to consider for these points is how the routes will be generated - e.g, is there an entrance where the itinerary ends, or will the user have to walk around the building to find one?  

For now, you can probably just check the commit history for the ones that you changed.